SCons/Environment.xml

```diff
@@ -383,12 +392,15 @@ AddPostAction(foo, Chmod('$TARGET', "a-x"))
 </arguments>
 <summary>
 <para>
-Arranges for the specified
+Arrange for the specified
 <parameter>action</parameter>
 to be performed
 before the specified
 <parameter>target</parameter>
 is built.
+<parameter>target</parameter>
```

Comment:
Any reason to only add this clarification here?
Seems like this is always true for all builders?

Reply:
`AddPostAction` and `AddPreAction` aren't builders, So I figured it was less obvious for these two.